src/common/ceph_argparse.cc

```diff
@@ -241,7 +241,7 @@ bool parse_ip_port_vec(const char *s, vector<entity_addrvec_t>& vec, int type)
 
 // The defaults for CephInitParameters
 CephInitParameters::CephInitParameters(uint32_t module_type_)
-  : module_type(module_type_)
+  : module_type(module_type_), no_config_file("false")
```

Comment:
``"false"`` evaluates to ``true``.. and in any case the header sets the default value

Reply:
@liewegas fixed and repushed.